docs/_static/css/custom.css

```diff
@@ -1,3 +1,14 @@
+/* Add additional spacing between sections using semantic section elements, until we update sphinx-wagtail-theme */
+section {
+  margin-top: 2.5rem;
+  margin-bottom: 2.5rem;
+}
+
+/* Workaround for https://github.com/executablebooks/MyST-Parser/issues/533 */
+li > p {
+  margin-bottom: 0;
+}
+
```

Comment:
We should probably adopt MyST Parser in our Wagtail Sphinx theme.

This way we can triage and add core styling for these issues.

https://github.com/executablebooks/MyST-Parser/issues/533
https://github.com/executablebooks/MyST-Parser/issues/534

Reply:
https://github.com/wagtail/sphinx_wagtail_theme/issues/133 created